### ISSUE TITLE
Pandas in built vis - Missing file

### ISSUE THREAD

**Issue by ameachasag** [HiPyLiv/HiPyProject#9]
The file "countries.xlsx" needed in the worked example is missing so a FileNotFoundError comes up when trying:
```python
countries = pd.read_excel("countries.xlsx", index_col="Country")
print(countries.head()) 
```

**Comment by stmball** [comment]
Fixed!

**Comment by hipylivadmin** [comment]
Fixed in 7894e2f9077d3b13340f43d1d6986ef57ea31190
